_posts/0000-01-02-anuragjul84.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
------
layout: slide
title: "Welcome to our second slide!"
---